changelog.d/20241002_125030_30907815+rjmello_compute_client.rst

```diff
@@ -0,0 +1,8 @@
+Added
+~~~~~
+
+- Added an initial Globus Compute client class, ``globus_sdk.ComputeClient``. (:pr:`NUMBER`)
+  - ``globus_sdk.ComputeAPIError`` is the error class for this client.
+  - ``ComputeClient.get_function`` is a method to get information about a registered function.
+  - Users can access scopes for the Globus Compute API via ``globus_sdk.scopes.ComputeScopes`` or
+    ``globus_sdk.ComputeClient.scopes``.
```

Comment:
rst is a bit finnicky; this [doesn't render as desired](https://github.com/globus/globus-sdk-python/blob/b1d69ba2cfd7fea26385aa4c3e70f0c1a4165fdf/changelog.d/20241002_125030_30907815%2Brjmello_compute_client.rst)

```suggestion
-    Added an initial Globus Compute client class, ``globus_sdk.ComputeClient``. (:pr:`NUMBER`)

     -    ``globus_sdk.ComputeAPIError`` is the error class for this client.
     -    ``ComputeClient.get_function`` is a method to get information about a registered function.
     -    Users can access scopes for the Globus Compute API via ``globus_sdk.scopes.ComputeScopes`` or
    ``globus_sdk.ComputeClient.scopes``.
```

Reply:
Ah, good catch. I just pushed an update.